### DISCUSSION TITLE
python class not found on import

### DISCUSSION THREAD

**Discussion by sfc-gh-tlarkin** [CrowdStrike/falconpy#374] [Q&A] [answered]
**Describe the bug**
Using the examples from the docs I am trying to import sensor_update_policies

**To Reproduce**
In the Python shell with falconpy installed via pip
>>> from falconpy import sensor_update_policies as FalconSUP
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'sensor_update_policies' from 'falconpy'

**Expected behavior**
the class is imported

**Environment (please complete the following information):**
 - OS: macOS and Amazon Linux 2
 - Python 3.8.5

**Comment by jshcodes** [comment, marked as answer]
If you are importing the module file like displayed above, you will need to use the singular name.
```python
from falconpy import sensor_update_policy as FalconSUP
falcon = FalconSUP(client_id="ID_HERE", client_secret="SECRET_HERE")
```

You can also just import the class directly if you are using v0.7.0+.
```python
from falconpy import SensorUpdatePolicy
falcon = SensorUpdatePolicy(client_id="ID_HERE", client_secret="SECRET_HERE")
```
Starting in v0.7.1+ you can also use the plural alias.
```python
from falconpy import SensorUpdatePolicies
falcon = SensorUpdatePolicies(client_id="ID_HERE", client_secret="SECRET_HERE")
```

**Comment by t-lark** [comment]
ah thank you, and yeah I noticed it was plural vs singular (posting from my personal gh account)
